docs/src/index.md

```diff
@@ -492,7 +492,97 @@ end
 end
 ```
 
-# Working with `Symbol`s without referring to columns
+## Constructing multi-column arguments and `src => fun => dest` calls using `$`
+
+If an argument is entirely wrapped in `$()`, the result bypasses the anonymous function creation of DataFramesMeta.jl and is passed to the underling DataFrames.jl function directly. Importantly, this allows for multi-column selection and passing `src => fun => dest` calls from the DataFrames.jl "mini-language" directly.
+
+### Multi-argument selectors 
+
+To refer to multiple columns in DataFrames.jl, one can write
+
+```julia
+select(df, [:a, :b])
+```
+
+which selects the columns `:a` and `:b` in the data frame. We can generate this command in DataFramesMeta.jl with
+
+```julia
+@select df $[:a, :b]
+```
+
+Similarly, to selct all columns beginning with the letter `"a"`, wrap a regular expression in `$()`. 
+
+```julia
+@select df $(r"^a")
+```
+
+which will construct the command `select(df, r"^a")`. 
+
+Multi-argument selectors *may only* be used when an entire argument is wrapped in `$()`. For example
+
+```julia
+@select df :y = f($[:a, :b])
+```
+
+will fail. 
+
+Not all functions in DataFrames.jl allow for multi-column selectors, so detailed knowledge of the underlying functions DataFrames.jl may be required. For example, the call 
+
+```julia
+subset(df, [:a, :b])
```

Comment:
we potentially could allow multi-column selectors in `subset`, but I thought they would not be very useful in practice and the code would be more complicated. Do you think they should be added?

Reply:
No, I don't think it should be added. It's not intuitive that the columns should be "and"-ed.

Reply:
OK. So we are on the same page.